src/components/Admin/ManageSensors/ManageSensors.tsx

```diff
@@ -79,32 +79,28 @@ const ManageSensors: () => JSX.Element = () => {
           boxShadow="lg"
           textAlign="center"
         >
-          <Heading marginY={2}>{t('manageSensors')}</Heading>
+          <Heading marginY={2}>{t('heading')}</Heading>
           <Grid
             justifyContent="center"
-            templateColumns={['repeat(2,1fr)', null, 'repeat(4,1fr)', null]}
+            templateColumns={[
+              'repeat(1,1fr)',
+              'repeat(2,1fr)',
+              'repeat(3,1fr)',
+              'repeat(6,1fr)',
+            ]}
             gap={2}
           >
+            <ToggleActiveModal sensors={inactiveSensors} isActive={false} />
             <AddSensorModal />
             <DownloadCSVModal sensors={sensors} />
-            <DeleteOldDataModal />
+            <ToggleActiveModal sensors={activeSensors} isActive />
```

Comment:
Didn't know you could do `isActive` instead of `isActive={true}`. Maybe use the long form for readability?